test/crypto_kem/nistkat.c

```diff
@@ -38,7 +38,9 @@ static void fprintBstr(FILE *fp, const char *S, const uint8_t *A, size_t L) {
     fprintf(fp, "\n");
 }
 
-int main() {
+int main(int argc, char** argv) {
```

Comment:
Wouldn't the following suffice?

```suggestion
int main(void) {
```

Reply:
Yeah I keep being confused if that's okay or not... But we have it in a bunch of different places as well I suppose.